.ci/Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Include the VM runner specific targets.
+include job/vm/Makefile.include
+
+.PHONY: help
+help:
+	@echo "Job related targets"
+	@echo ""
+	@echo "  job-run-NAME             - Run the job NAME. Use KATA_TESTS_JOB_RUNNER to set the runner."
+	@echo "  job-list                 - List all available CI jobs"
+	@echo "help: CI targets"
+	@echo "  vm-help                  - Help for the VM runner targets."
+	@echo "Environment variables:"
+	@echo "  KATA_TESTS_JOB_RUNNER:   - local or vm (default)"
+
```

Comment:
The new interface does look a lot nicer.  A couple of overall concerns:
 - Can the docs be updated to describe this new preferred way to run tests
 - Adding a few more hundred lines of fairly complex bash to the thousands of existing lines of fragile bash code makes me kinda nervous.  But maybe it's a necessary interim step.